File: app/src/main/java/com/candy/capture/receiver/CaptureReceiver.java
package com.candy.capture.receiver;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;
import android.util.Log;

import com.candy.capture.core.SharedPreferenceManager;
import com.candy.capture.service.FloatingWindowService;

public class CaptureReceiver extends BroadcastReceiver {
    private static final String TAG = "CaptureReceiver";

    public CaptureReceiver() {
    }

    @Override
    public void onReceive(Context context, Intent intent) {
        Log.d(TAG, "onReceive action:" + intent.getAction());
//        TipsUtil.showToast(context, "onReceive action:" + intent.getAction());
        if (SharedPreferenceManager.getInstance(context).isAllowFastCapture()) {
            Intent intent1 = new Intent(context, FloatingWindowService.class);
            context.startService(intent1);
        }
    }
}

File: app/src/main/java/com/candy/capture/receiver/CaptureReceiver.kt
package com.candy.capture.receiver

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent
import android.util.Log
import com.candy.capture.core.SharedPreferenceManager
import com.candy.capture.service.FloatingWindowService

/**
 * @Description
 * @Author zhanghaiqiang
 * @Date 2020/9/30 15:27
 */
class CaptureReceiver: BroadcastReceiver() {
    private val TAG = "CaptureReceiver"

    override fun onReceive(context: Context, intent: Intent) {
        Log.d(TAG, "onReceive action:" + intent.action)
        if (SharedPreferenceManager.getInstance(context).isAllowFastCapture()) {
            val intent1 = Intent(context, FloatingWindowService::class.java)
            context.startService(intent1)
        }
    }
}